### PR TITLE
Add Next Tutorial

### PR DESCRIPTION
**Pull request by MaxStrange**
This tutorial shows you how to:

1. Train a PyTorch model in AML (not really the focus of the tutorial)
2. Convert the model to OpenVINO
3. Double check it still works
4. Create a G-API graph for your custom model and run it on your PC
5. Transition that graph to the Percept DK and run the custom model there

Two things to note:

1. The neural network is not great. I have not tuned it yet, but I can do that in my downtime. I don't want to block getting the tutorial up waiting for me to tune the network.
2. I recommend people read the first tutorial before tackling this one.

### CODE REVIEW

**Comment by tony979** [comment]
Could you fix the typo in azureeyemodule/README.md?

File: azureeyemodule/README.md [hunk 1]
```diff
@@ -308,7 +308,7 @@ Either way, once you have buildx installed, run the following to cross compile:
 `docker buildx build --platform linux/arm64 --tag azureeyemodule-xc -f Dockerfile.arm64v8 --load .`, which will build the azureeyemodule Docker image,
 and with it, the Percept application.
 
-Once you have your new azureeyemodule, stick it into any container registry and pull it using the module twin.
+Once you have your new azureeyemodule, stick it into any container registry and pull it using the IoT Hubt.
```
Comment (Devinwong) [inline azureeyemodule/README.md:311]: typo

**Comment by Devinwong** [comment]
I think you really did a great job, Max. You definitely spent a lot of time delivering this tutorial and the sample codes. This is very helpful for advanced developer to bring a new model in.

One question, how do we want to maintain the mock-eye-module? Most of the codes are similar to the original one. When there is new change to the original one, are we going to copy/paste to the mock one? Or we just keep the mock one as is? Thanks.

File: mock-eye-module/README.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
+# Mock Eye Module
```
Comment (tony979) [inline mock-eye-module/README.md:1]: Copyright?

**Review by Devinwong — CHANGES_REQUESTED** [review]
I think you really did a great job, Max. You definitely spent a lot of time delivering this tutorial and the sample codes. This is very helpful for advanced developer to bring a new model in.

One question, how do we want to maintain the mock-eye-module? Most of the codes are similar to the original one. When there is new change to the original one, are we going to copy/paste to the mock one? Or we just keep the mock one as is? Thanks.

File: mock-eye-module/README.md (hunk 2 above)
Comment (MaxStrange) [reply mock-eye-module/README.md:1]: We don't have copyrights in the READMEs that I am aware of, but I did forget to add a couple notes here. I'll fix that.

**Comment by MaxStrange** [comment]
Good question. I didn't want to include the files directly from the azureeyemodule because it would make it more confusing for people who are using the mock eye module as an example, but you are right that it leads to code duplication.

I am okay with the code being updated infrequently (only as we update the OpenVINO version), and have it fall behind the code in the azureeyemodule, since the point is simply to serve as a sand box for the G-API, and it doesn't need to be exactly the same as the azureeyemodule.

Thoughts?

**Review by Devinwong — APPROVED** [review]

**Pull request merged**